frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/utilization-card/utils.tsx

```diff
@@ -9,11 +9,10 @@ import {
 
 export const humanizeIOPS: Humanize = (value) => {
   const humanizedNumber = humanizeNumber(value);
-  const unit = 'IOPS';
   return {
     ...humanizedNumber,
-    string: `${humanizedNumber.value} ${unit}`,
-    unit,
+    string: `${humanizedNumber.string} IPOS`,
```

Comment:
Do string and unit both need to have IOPS unit?

Reply:
yes. Or else the two would not be consistent.
`value + unit === string` ( Format for humanize functions)